src/modules/credentials/handlers/AutoResponseHandler.ts

```diff
@@ -0,0 +1,179 @@
+import type { Logger } from '../../../logger/Logger'
```

Comment:
Few last points:
- I think we shouldn't call this a handler as is doesn't handle the auto response
- `handlers/` directory is for message handlers
- If you inject the agent config and logger in the constructor (using DI) you don't need to pass these to the methods. I think this is a bit cleaner

Reply:
Also, please update the `issueCredential` method in `src/__tests__/helpers.ts` to use auto issuance.